web/app/Models/User.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Wikijump\Models;
+
+use Illuminate\Contracts\Auth\MustVerifyEmail;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Foundation\Auth\User as Authenticatable;
+use Illuminate\Notifications\Notifiable;
+
+class User extends Authenticatable
+{
+    use HasFactory, Notifiable;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'name',
+        'email',
+        'password',
+    ];
+
+    /**
+     * The attributes that should be hidden for arrays.
```

Comment:
What does "hidden for array" mean?

Reply:
When you work with a user object later, you can assume you don't want the password field or the remember token returned in the selection. This can be overridden if needed but it's a handy way to secure secrets for most use cases.